### DISCUSSION TITLE
TimeZoneDisplayName

### DISCUSSION THREAD

**Discussion by shadowpuck99** [tomasrudh/analogclock#18] [Q&A]
i'm sure i'm missing something quite simple - but, any thoughts on why i cannot get the timezonedisplayname to show?
everything else i've tried seems to be working (meaning - i can hide various elements, change the colors, etc.) but, even though i've set a timezonedisplayname it's not showing......

**Comment by tomasrudh** [comment]
Hi, I seem to have missed to state in the documentation that you have to set show_timezone: true for the time zone to show at all, then you should be able to have timezonedisplayname working. Let me know how that goes.

/Tomas

**Reply by shadowpuck99** [reply]
hi!  thank you for the suggestion.
i just tried it and still no success.  i'm going to try and make sure this isn't a browser cache issue or something.....

here's what i'm using to create the card:
type: 'custom:analog-clock'
hide_secondhand: false
hide_weekday: true
hide_digitaltime: true
show_timezone: true
color_text: Black
color_secondhand: Red
color_hourhand: '#326ba8'
color_minutehand: '#3273a8'
hide_minorticks: true
timezone: America/Chicago
timezonedisplayname: Central

**Reply by shadowpuck99** [reply]
i just tried a different computer and browser and still not showing the timezone.....very strange!

**Reply by shadowpuck99** [reply]
it will show the weekday, and digital time.

**Reply by tomasrudh** [reply]
Try to set hide_weekday: false. This is probably a bug I should take a look at.

**Reply by shadowpuck99** [reply]
so, yes, changing hide_weekday: false and setting show_timezone: true with timezonedisplayname set did indeed show the timezone label.

oh my.  that was about the only thing i didn't think to try!  :)
i can totally understand why it acted that way.

**Reply by tomasrudh** [reply]
Great! So, the history is that this field only show week day from the beginning, hence the hide_weekday setting. Later I added time zone to the same position. I will see if I can adjust, and / or document this to be more clearer.

Thanks for letting me know!

/Tomas

**Reply by shadowpuck99** [reply]
makes total sense now - i just didn't think to try that!
just having that documented will be a huge help if it can't be adjusted for some reason.  

thank you for the help and for developing this - it's a neat little thing to use in home assistant.
